### DISCUSSION TITLE
Unable to login to npmjs.com

### DISCUSSION THREAD

**Discussion by xdumstr** [npm/feedback#809] [Website]
I am unable to log into npmjs.com.  After entering the OTP code. I am redirected to this 404 Page.  Message says "Route not found!".  

![Screen Shot 2022-11-15 at 11 55 36 AM](https://user-images.githubusercontent.com/79477922/201980264-291b3a47-808d-4139-8047-21c763d268de.png)

I have retried a few times, and getting the same error. Can you help?

**Comment by xylesoft** [comment]
Second, I also get this issue and cannot log in to my account via the website. Although the npm login (CLI) does work.

**Reply by xylesoft** [reply]
I resolved the issue by using `npm profile enable-2fa` to set up two-factor authentication and logging in with a 2FA OTP. 

Additionally, `npm login` worked using the email OTP.
